docs/io/output/progress_bars.rst

```diff
@@ -0,0 +1,11 @@
+********************************
+Progress Bars for Simulation Run
+********************************
+
+TARDIS displays progress bars by default to track the simulation's progress.
+
+.. image::
+    ../images/progress_bars_demo.gif
+    :width: 600
+    
```

Comment:
Thank you for making this change! Could this be just a little bit larger? Just enough to make words clear in the default view?